terraform/rds.tf

```diff
@@ -26,6 +26,28 @@ resource "aws_db_instance" "wfnews_pgsqlDB" {
   enabled_cloudwatch_logs_exports = ["postgresql"]
   parameter_group_name            = "wfnews-manual"
 }
+
+resource "aws_db_instance" "wfone_pgsqlDB" {
```

Comment:
We'll have to verify the DB configuration once we deploy the notification DB updates. Ths will likely be a schema in the existing DB rather than a new DB?

Reply:
Commented out for now and used the existing datasource config for push notifications, will revisit when db is deployed